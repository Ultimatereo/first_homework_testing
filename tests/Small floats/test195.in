3
4.57e-07 6.1e-08 2.71e-07 1.3e-07
2.5e-07 3.76e-07 3.96e-07 1e-07
3.79e-07 7.4e-08 2.47e-07 8.8e-08
