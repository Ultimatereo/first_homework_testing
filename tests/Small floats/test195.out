2.30423
3.04231
-4.09082
